api-reference/beta/api/device_update.md

```diff
@@ -29,7 +29,6 @@ In the request body, supply the values for relevant fields that should be update
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
 |accountEnabled|Boolean| **true** if the account is enabled; otherwise, **false**. |
-|alternativeSecurityIds|[alternativeSecurityId](../resources/alternativesecurityid.md) collection| The **any** operator is required for filter expressions on multi-valued properties. Not nullable.           |
```

Comment:
Definitely remove this, but more than that - I'm not sure that a lot of these can be edited by an app/admin.  Can you check with Shantaram, Jairo and Billy please?